main.yaml

```diff
@@ -105,19 +105,21 @@
 
   - name: Refresh node facts after floating IP attach
     os_server_facts:
-      server: '*'
+      server: 'kolla-*'
+
+  - debug:
+      var: openstack_servers
 
   - name: Add all nodes to the inventory
     add_host:
-      name: "{{item.server.public_v4}}"
+      name: "{{item.public_v4}}"
       groups: all
       ansible_user: "{{ansible_user}}"
-      instance_name: "{{item.server.name}}"
+      instance_name: "{{item.name}}"
       ansible_ssh_private_key_file: "{{key}}"
       ansible_ssh_common_args: -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no
     with_items:
-      - "{{kolla_nodes.results}}"
-      - "{{kolla_controllers.results}}"
+      - "{{openstack_servers}}"
 
```

Comment:
since openstack_servers should include the `operator` node, that node should be already part of the `all` group and we probably don't need to add it again on line 127